src/views/PreOrder.vue

```diff
@@ -318,15 +322,33 @@ export default defineComponent({
       return this.getTemporalExpr(this.getJobStatus(this.jobEnums[enumId]))?.description ?
         this.getTemporalExpr(this.getJobStatus(this.jobEnums[enumId]))?.description :
         this.$t('Disabled')
+    },
+    async getMorePreOrderJobs() {
+      await this.store.dispatch("job/fetchMoreJobs", {
+        "inputFields":{
+          "enumTypeId": "MISC_SYS_JOB",
```

Comment:
Update enumTypeId as per the page on which we are fetching more jobs.

Reply:
Done sir